README.md

```diff
@@ -55,6 +56,27 @@ Note: grab-site currently **does not work with Python 3.5**; please use Python 3
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+Install with Docker
+---
+Get the pre-built docker container:
```

Comment:
Try, maybe just point to https://docs.docker.com/engine/installation/ if you consider that accurate.